============================= test session starts ==============================
platform linux -- Python 3.12.7, pytest-8.3.3, pluggy-1.5.0
django: version: 4.2.9, settings: mysite.settings.local (from env)
rootdir: /root/app
configfile: pytest.ini
plugins: django-4.9.0, srcpaths-1.2.1, cov-5.0.0
collected 16 items

appxxx/tests/test_models.py ........                                     [ 50%]
appxxx/tests/test_serializers.py ...                                     [ 68%]
appxxx/tests/test_views.py .....WARNING: Failed to generate report: No data to report.

                                                                         [100%]

------------------- generated xml file: /root/app/pytest.xml -------------------

---------- coverage: platform linux, python 3.12.7-final-0 -----------

============================== 16 passed in 1.21s ==============================
